mailpoet/lib/Subscription/Throttling.php

```
@@ -63,7 +63,7 @@ public function throttle() {
 
   public function purge(): void {
     $interval = $this->wp->applyFilters('mailpoet_subscription_purge_window', MONTH_IN_SECONDS);
-    $this->subscriberIPsRepository->deleteCreatedAtBeforeTimeInSeconds($interval);
+    $this->subscriberIPsRepository->deleteCreatedAtBeforeTimeInSeconds(is_int($interval) ? $interval : MONTH_IN_SECONDS);
```

PHPStan (2.1.51)

[error] Constant MONTH_IN_SECONDS not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)